homeassistant/components/auth/login_flow.py

```diff
@@ -199,6 +199,7 @@ async def _async_flow_result_to_response(
         if result["type"] != data_entry_flow.FlowResultType.CREATE_ENTRY:
             # @log_invalid_auth does not work here since it returns HTTP 200.
             # We need to manually log failed login attempts.
```

Comment:
There is a comment here that is no longer correct and needs to be addressed.

Reply:
I removed the outdated comment and added a short comment about the new behaviour, I hope it fits your expectations!